packages/yswaps/scripts/libraries/solvers/multicall/CurveSpellEth.ts

```diff
@@ -131,14 +158,14 @@ export class CurveSpellEth implements Solver {
           _tokenIn: this.crvAddress,
           _tokenOut: this.crvSpellEthAddress,
           _amount: crvBalance,
-          _minAmountOut: BigNumber.from('0'),
+          _minAmountOut: minAmountOut,
```

Comment:
is it correct to use same `minAmountOut` for both trades? `minAmountOut` is using the total amount of weth after swaping both tokenIns to weth.

Reply:
`minAmountOut` here refers to `crvSpellEth` balance as far as I see it.